### PR TITLE
Fix/forms 992

### PR DESCRIPTION
**Pull request by nimya-aot**
# Description
Created new file **forms-settings.cy.js** as part of updating existing cypress test scripts and moved all the working test scripts files to a newly created folder named as  **e2e** 

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have approval from the product owner for the contribution in this pull request

### CODE REVIEW

File: tests/functional/cypress/e2e/form-settings.cy.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    });
+  });
+    
+// Verifying fields in the form settings page
+ it('Visits the form settings page', () => {
+   
+    cy.visit(`/${depEnv}/form/create`);
+    cy.location('pathname').should('eq', `/${depEnv}/form/create`);
+    cy.contains('h1', 'Form Settings');
+    cy.get('.v-row > :nth-child(1) > .v-card > .v-card-title > span').contains('Form Title');
```
Comment (WalterMoar) [inline tests/functional/cypress/e2e/form-settings.cy.js:38]: I'm just curious: should we be putting `data-cy="form-settings-title"` (or similar) attributes on component elements?

Comment (nimya-aot) [reply tests/functional/cypress/e2e/form-settings.cy.js:38]: @WalterMoar No, Its not putting by us. These attributes are from common components.

File: tests/functional/cypress/specs/floating-menu-buttons-form-design.cy.js [hunk 2]
```diff
@@ -12,22 +12,30 @@ Cypress.Commands.add('waitForLoad', () => {
 
 describe('Form Designer', () => {
   beforeEach(()=>{
+    debugger;
     cy.viewport(1000, 1800);
     cy.waitForLoad();
     cy.kcLogout();
```
Comment (jasonchung1871) [inline tests/functional/cypress/specs/floating-menu-buttons-form-design.cy.js:18]: Is there a particular reason why cy.kcLogout gets removed everywhere else but not here?

Comment (nimya-aot) [reply tests/functional/cypress/specs/floating-menu-buttons-form-design.cy.js:18]: @jasonchung1871 This file is not working anymore, so please ignore this. I created a new folder **e2e**. I moved all the files I updated /created to that folder. Cypress tests will run from that folder. Could you please check that?

**Review by jasonchung1871 — APPROVED** [review]

**Pull request merged**